src/services/mockCategoriesService.js

```diff
@@ -0,0 +1,27 @@
+import dishes from '../assets/data/dishes.json';
+import categories from '../assets/data/categories.json';
+export default class CategoryService {
```

Comment:
Create and empty line between line 2 and three

Reply:
Resolved